docs/README.md

```diff
@@ -1,43 +1,103 @@
-## Getting Help
+# Metabase documentation and resources
 
-### [Troubleshooting Guide](troubleshooting-guide/index.md)
+## Tutorials
+
+### [Learn Metabase][learn]
+
+Learn how to set Metabase up, build out your analytics, organize things and set permissions, and how to do data ops right.
+
+## Getting help
+
+### [Troubleshooting Guide][troubleshooting]
 
 Problems, their causes, how to detect them, and how to fix them.
 
-### [FAQs](faq/start.md)
+### [Metabase forum][forum]
+
+A place to get help on installation, setting up as well as sharing tips and tricks.
+
+### [FAQs][faq]
 
 Frequently asked questions about Metabase.
 
-## Metabase guides
+## Metabase reference guides
 
-### [Users Guide](users-guide/start.md)
+Documentation guides for the Metabase application.
+
+### [Users Guide][users-guide]
 
 How to ask questions, how to visualize answers, as well as how to share questions and create dashboards.
 
-### [Admin Guide](administration-guide/start.md)
+### [Admin Guide][admin-guide]
 
 How to set up Metabase, configure common settings, manage accounts and permissions, and add databases.
 
-### [Operations Guide](operations-guide/start.md)
+### [Operations Guide][operations-guide]
 
 Learn how to install Metabase for production use. The guide covers topics like SSL termination, deploying via Docker Containers vs. JAR files, as well as the tradeoffs involved.
 
-### [Enterprise Guide](./enterprise-guide/start.md)
+### [Enterprise Guide][enterprise]
 
-Here’s where to go for help using the features included in [Metabase Enterprise Edition](https://www.metabase.com/enterprise/).
+Here’s where to go for help using the features included in [Metabase Enterprise Edition][enterprise-landing].
 
-## For developers
+## Metabase for developers
 
-### [Developers Guide](developers-guide/start.md)
+### [Developers Guide][developers]
 
-Learn how to contribute to the Metabase open source project. 
+Learn how to contribute to the Metabase open source project.
 
-### [Driver Development](developers-guide-drivers.md)
+### [Driver Development][drivers]
 
 This guide lists existing community drivers, and shows how to get started with driver development.
 
+### [Embedding reference apps][embedding-ref-apps]
+
+Code examples for embedding Metabase in
```

Comment:
maybe there's a period missing or we have to say "embedding Metabase in your application"